src/overlays/actors/ovl_En_Bomjima/z_en_bomjima.c

```diff
@@ -74,33 +74,20 @@ static ColliderCylinderInit sCylinderInit = {
     { 10, 30, 0, { 0, 0, 0 } },
 };
 
-static u16 D_80C00A44[] = {
-    0x0719,
-    0x071A,
-    0x071B,
-    0x0708,
-};
-
-static u16 D_80C00A4C[] = {
-    0x0739,
-    0x073A,
-    0x073B,
-    0x0000,
-};
+u16 D_80C00A44[] = { 0x0719, 0x071A, 0x071B, 0x0708 };
 
-static u16 D_80C00A54[] = { 0x0739, 0x073A, 0x073B, 0x0714, 0x0709, 0x070A, 0x070B,
-                            0x070C, 0x070D, 0x070E, 0x070F, 0x0712, 0x0713 };
+u16 D_80C00A4C[] = { 0x0739, 0x073A, 0x073B, 0x0000 };
 
-static u16 D_80C00A70[] = { 0x0739, 0x073A, 0x073B, 0x0759, 0x0753, 0x0754, 0x0755,
-                            0x0756, 0x070D, 0x0757, 0x0758, 0x0712, 0x0713 };
+u16 D_80C00A54[] = {
+    0x0739, 0x073A, 0x073B, 0x0714, 0x0709, 0x070A, 0x070B, 0x070C, 0x070D, 0x070E, 0x070F, 0x0712, 0x0713,
+};
```

Comment:
Could you get rid of the leading 0's in all this data above and below?

Reply:
I'm guessing these are textids, the first couple make sense in context in a text dump I just checked. We probably do want to lose the leading 0s, though.